packages/flutter/lib/src/material/theme_data.dart

```diff
@@ -334,6 +335,8 @@ class ThemeData with Diagnosticable {
     TimePickerThemeData? timePickerTheme,
     ToggleButtonsThemeData? toggleButtonsTheme,
     TooltipThemeData? tooltipTheme,
+    // ANIMATION THEME
```

Comment:
would stick this in GENERAL CONFIGURATION above, as the first element (because alphabeticacally)

Reply:
reordered